delete/wp-site

```diff
@@ -9,8 +9,11 @@ fi
 
 cd "$(dirname $0)"
 # Read config file with paths to WP-installs and usernames
-source ../conf
+source /usr/local/default/etc/wp-mass-admin/wpma.conf
 
+if [[ -e /usr/local/etc/wp-mass-admin/wpma.conf ]]; then
+    source /usr/local/etc/wp-mass-admin/wpma.conf
+fi
```

Comment:
1. Default conf is same as the empty "regular" right now
2. Non-default
3. Probably not. Changing behaviour.